src/ui/components/filters/geolocationcomponent.js

```diff
@@ -264,7 +265,7 @@ export default class GeoLocationComponent extends Component {
     this.setState({ geoError: true });
     const { enabled, message } = this._geolocationTimeoutAlert;
     if (enabled) {
-      window.alert(message);
+      alertify.alert(message);
```

Comment:
Can we wrap the library's alert function inside our own alert() helper function? That way our code doesn't depend directly on alertify, and if we need to swap out the alert library, we can make the change in a single file

Reply:
will do!